install/host_preparation.adoc

```diff
@@ -17,14 +17,33 @@ ifdef::openshift-enterprise[]
 [[software-prerequisites]]
 == Operating System Requirements
 
-A base installation of 7.4 or 7.5 (with the latest packages from the Extras
-channel) or RHEL Atomic Host 7.4.2 or later is required for master and node
-hosts. See the following documentation for the respective installation
+The operating system requirements for master and node hosts are different
+depending on your server architecture.
+
+* For servers that use x86_64 architecture, use a base installation of Red Hat
+Enterprise Linux (RHEL) 7.4 or 7.5 with the latest packages from the Extras
+channel or RHEL Atomic Host 7.4.2 or later.
+* For cloud-based installations, use a base installation of RHEL 7.4 or 7.5 with
+the latest packages from the Extras channel.
+* For servers that use IBM POWER8 architecture, use a base installation of RHEL
+7.5 with the latest packages from the Extras channel.
+* For servers that use IBM POWER9 architecture, use a base installation of
+RHEL-alt 7.5 with the latest packages from the Extras channel.
+
+See the following documentation for the respective installation
 instructions, if required:
 
 - https://access.redhat.com/documentation/en-US/Red_Hat_Enterprise_Linux/7/html/Installation_Guide/index.html[Red Hat Enterprise Linux 7 Installation Guide]
 - https://access.redhat.com/documentation/en/red-hat-enterprise-linux-atomic-host/version-7/installation-and-configuration-guide/[Red Hat Enterprise Linux Atomic Host 7 Installation and Configuration Guide]
 
+[[server-type]]
+== Server Type Requirements
+
+If you use IBM POWER servers for your nodes, you can use only IBM POWER servers.
+You cannot add nodes that run on IBM POWER servers to an existing cluster that
+uses x86_64 servers or deploy cluster nodes on a mix of IBM POWER and x86_64
+servers.
```

Comment:
This should also apply to the Masters or control infrastructure. Based on what we have been told in CEE